hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/props/MirrorNetworkStake.java

```diff
@@ -0,0 +1,38 @@
+package com.hedera.mirror.test.e2e.acceptance.props;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import lombok.Data;
+
+@Data
+public class MirrorNetworkStake {
+    private long maxStakingRewardRatePerHbar;
+    private int nodeRewardFeeDenominator;
+    private int nodeRewardFeeNumerator;
```

Comment:
This should match the REST API, not the domain model. Same for other fields. I don't think anything is an int32.
```suggestion
    private int nodeRewardFee;
```

Reply:
Thanks I have changed these to match the REST API. The denominators and numerators have been removed.